src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/oasyscontext/AssessmentInfo.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.model.oasyscontext
+
+import java.time.LocalDateTime
+
+abstract class AssessmentInfo(
+  val assessmentId: Long,
+  val assessmentType: String,
+  val dateCompleted: LocalDateTime?,
```

Comment:
I think the way the upstream service works is that the `dateCompleted` property will be missing from the response if it isn't set.

Reply:
I'm not sure but I think the integrations team receive dates without any zone info and will be adding it to return dates in the form `"2022-11-09T14:33:53Z"`.

This conforms with the policy [ADR Use Zoned Datetimes](https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/doc/adr/0003-use-zoned-datetimes.md)

As we are actually only interested in the dates and not the times is it "easy" to flexibly handle these iso8601 formats to discard the time information and pass on the date only e.g. `2022-12-25`?

Reply:
Makes sense - I've adjusted to `OffsetDateTime` then and updated the stubs to add an offset.